src/redux-enhancers/undoRedoReducerEnhancer.ts

```diff
@@ -212,6 +212,20 @@ const undoRedoReducerEnhancer: StoreEnhancer<any> =
         return newState
       }
 
+      // if an undoable action is of type importText, update inversePatches adding latest patch of actions which contains accurate patches for importText action
```

Comment:
This comment is good, but it needs to go a little further. The key question we need to answer for future developers is: Why does `importText` require a special case? What is different from the normal case? What would happen if only the normal case was used?

Reply:
What's the best way to add long descriptive comments @raineorshine ?

Reply:
It doesn't have to be that long. Just 1-2 additional sentences. Try to answer the questions posed above.